pkg/oc/cli/cmd/set/env.go

```diff
@@ -124,8 +128,12 @@ func NewCmdEnv(fullName string, f *clientcmd.Factory, in io.Reader, out, errout
 		Long:    envLong,
 		Example: fmt.Sprintf(envExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
-			kcmdutil.CheckErr(options.Complete(f, cmd, args))
-			err := options.RunEnv(f)
+			options.Complete(f, cmd, args)
+			err := options.Validate(cmd, args)
+			if err != nil {
+				kcmdutil.CheckErr(err)
+			}
+			err = options.RunEnv(f)
 			if err == kcmdutil.ErrExit {
```

Comment:
Since you're touching it, would you mind fixing this to follow the above pattern as well. This check is not relevant, since it's already being performed by `kcmdutil.CheckErr` properly. Right @juanvallejo ?

Reply:
done